src/bin/bench.rs

```diff
@@ -50,20 +50,31 @@ pub fn execute(options: Options, config: &Config) -> CliResult<Option<()>> {
     let root = try!(find_root_manifest_for_cwd(options.flag_manifest_path));
     config.shell().set_verbose(options.flag_verbose);
 
+    let mut benches = Vec::new();
```

Comment:
this vector looks unnecessary - seems you could use `options.flag_bench.as_slice()`